lib/navigation/crosscheck.cpp

```diff
@@ -0,0 +1,52 @@
+#include "crosscheck.hpp"
+
+namespace hyped::navigation {
+
+Crosscheck::Crosscheck()
+{
+  // TODO: implement
+}
+
+SensorChecks Crosscheck::checkTrajectoryAgreement(const core::ImuData imu_data,
+                                                  const core::EncoderData encoders_data,
+                                                  const core::KeyenceData keyence_data)
+{
+  /*
+  TODO: implement
```

Comment:
TODOLater for TODOs that are out of scope from this pr, otherwise the CI will flag it.

Reply:
Do we not want them all flagged? Then we know what still needs done before anything critical is run on this codebase? All my TODO comments will definitely need implemented before we run and isn't this the point of the TODO flag?